OBSDOCS-1462: Update monitoring config map API reference content for OCP 4.18 release (#88673)
openshift/openshift-docs: observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

|enableUserWorkload|*bool|`UserWorkloadEnabled` is a Boolean flag that enables monitoring for user-defined projects.

> |userWorkload|*link:#userworkloadconfig[UserWorkloadConfig]|`UserWorkload` defines settings for the monitoring of user-defined projects.

Reviewer: The asterisk is preexisting in many entries but does it relate to some note?

[options="header"]
|===
| Property | Type | Description

> |scrapeInterval|string|Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in: seconds (for example `30s`), minutes (for example `1m`) or a mix of minutes and seconds (for example `1m30s`). The default value is `30s`.

Reviewer: ```suggestion
|scrapeInterval|string|Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in seconds (for example `30s`), minutes (for example `1m`) or a mix of minutes and seconds (for example `1m30s`). The default value is `30s`.
```

Reviewer: Same for other entries.

> |evaluationInterval|string|Configures the default interval between rule evaluations in case the `PrometheusRule` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in: seconds (for example `30s`), minutes (for example `1m`) or a mix of minutes and seconds (for example `1m30s`). It only applies to `PrometheusRule` resources with the `openshift.io/prometheus-rule-evaluation-scope=\"leaf-prometheus\"` label. The default value is `30s`. |

Author: Oh my, and I even checked the rendering for this exact error :disappointed_relieved: Thank you for always chathing these :+1: it is fixed now

|thanosRuler|*link:#thanosrulerconfig[ThanosRulerConfig]|Defines the settings for the Thanos Ruler component in user workload monitoring.

|namespacesWithoutLabelEnforcement|[]string|Defines the list of namespaces for which Prometheus and Thanos Ruler in user-defined monitoring do not enforce the `namespace` label value in `PrometheusRule` objects.

> It allows to define recording and alerting rules that can query across multiple projects (not limited to user-defined projects) instead of deploying identical `PrometheusRule` objects in each user project.

Reviewer: ```suggestion
The `namespacesWithoutLabelEnforcement` property defines recording and alerting rules that can query across multiple projects (not limited to user-defined projects) instead of deploying identical `PrometheusRule` objects in each user project.
```

> It allows to define recording and alerting rules that can query across multiple projects (not limited to user-defined projects) instead of deploying identical `PrometheusRule` objects in each user project.

Reviewer 2: @eromanova97  I have to agree; there needs to be a noun here. Who or what is defining the rules?

Author: @mburke5678 I can make the change now, that should not cause issues I hope :crossed_fingers: , I might just make the change upstream after the release :+1:  if I do not sync in the meantime, it should be okay.

Author: I will create an issue to track this :+1:
